robot/resources/lib/python_keywords/epoch.py

```diff
@@ -33,15 +34,28 @@ def ensure_fresh_epoch(
     return epoch
 
 
+@allure.step("Align epochs for the whole cluster")
+@wait_for_success(60, 5)
+def align_epochs(shell: Shell, cluster: Cluster) -> bool:
```

Comment:
Shouldn't we add this to the tick_epoch call?

Reply:
I am not sure that it's necessary. I decided to add this check only for the cases when we are dealing with expired objects and epochs should be the same in the whole cluster.